templates/Dockerfile.template

```diff
@@ -1,3 +1,5 @@
+{{- $mongo4 := eq .Dir "4/debian9/4.0" -}}
```

Comment:
This file mostly uses tabs as an indents (instead of spaces). It could be great if we can follow this convention.

Reply:
done